libportal/meson.build

```diff
@@ -83,6 +87,21 @@ libportal_dep = declare_dependency(
   link_with: libportal,
 )
 
+if introspection
+  libportal_gir = gnome.generate_gir(libportal,
+    sources: generated_files + headers + src,
+    nsversion: gir_version,
+    namespace: 'Xdp',
+    symbol_prefix: 'xdp',
+    identifier_prefix: 'Xdp',
+    link_with: libportal,
+    includes: ['Gio-2.0'],
+    install: true,
+    install_dir_gir: girdir,
```

Comment:
I don't believe you need to specify the `install_dir*`'s unless they are custom locations?

Reply:
Good point. It's not necessary indeed.